client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-item.js

```diff
@@ -176,6 +195,7 @@ export class LabelItem extends Component {
 									{ this.renderPickup( carrierId ) }
 									{ this.renderRefund( labelId, refundExpired ) }
 									{ this.renderReprint( labelId, expired ) }
+									{ commercialInvoiceUrl && this.renderCommercialInvoiceLink( commercialInvoiceUrl ) }
```

Comment:
With the recent changes, it looks like this can just be 
```suggestion
									{ this.renderCommercialInvoiceLink( commercialInvoiceUrl ) }
```